tests/PropertyList.py

```diff
@@ -102,14 +102,14 @@ def testExists(self):
 
     def testGetVector(self):
         apl = dafBase.PropertyList()
-        v = (42, 2008, 1)
+        v = [42, 2008, 1]
         apl.setInt("ints", v)
-        apl.setInt("ints2", (10, 9, 8))
+        apl.setInt("ints2", [10, 9, 8])
```

Comment:
It would seem to be more pythonic to allow a tuple to be provided here (assuming the contents are always copied to the internal structure so mutability is not a problem). I'm happy to see how this turns out though. Do @ktlim or @TallJimbo have an opinion?